### DISCUSSION TITLE
Welcome to VinApp Discussions!

### DISCUSSION THREAD

**Discussion by williamvietnam** [williamvietnam/MyFirstAppAndroid#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by williamvietnam** [comment]
i changed app's name is WinWin. From now every one call app is WinWin instead of VinApp

**Comment by heclloor** [comment]
Hola
He instalado en un gps karoo la apk Vin-HKE y tengo problemas con las alarmas. Me repite las alarmas aunque las programe para no repetir, sobre todo la primera que pongo a los 30min. La repite cada 30 min
Puede ser un problema de la apk??
Puede ser que hago bien la programación??

gracias saludos
